ui/alarm.qml

```diff
@@ -0,0 +1,187 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.5
+import QtQuick.Layouts 1.3
+import Mycroft 1.0 as Mycroft
+import org.kde.kirigami 2.11 as Kirigami
+import QtGraphicalEffects 1.0
+
+Mycroft.Delegate {
+    id: root
+    skillBackgroundColorOverlay: "black"
+    property bool alarmExpired: sessionData.alarmExpired
+    property color alarmColor: alarmExpired ? "#FFFFFF" : "#22A7F0"
+    leftPadding: 0
+    rightPadding: 0
+    topPadding: 0
+    bottomPadding: 0
+    
+    Rectangle {
+        id: leftRect
+        anchors.top: parent.top
+        anchors.topMargin: Math.round(centerRect.height * 0.085)
+        anchors.right: centerRect.left
+        anchors.rightMargin: Math.round(-centerRect.width * 0.340)
+        color: alarmColor
+        radius: width
+        width: parent.width / 6
+        height: width
+        layer.enabled: true
+        layer.effect: OpacityMask {
+            maskSource: centerRect
+        }
+    }
+
+    Rectangle {
+        id: rightRect
+        anchors.top: parent.top
+        anchors.topMargin: Math.round(centerRect.height * 0.085)
+        anchors.left: centerRect.right
+        anchors.leftMargin: Math.round(-centerRect.width * 0.340)
+        color: alarmColor
+        radius: width
+        width: parent.width / 6
+        height: width
+        layer.enabled: true
+        layer.effect: OpacityMask {
+            maskSource: centerRect
+        }
+    }
+
+    Rectangle {
+        id: rightBottomRect
+        anchors.top: centerRect.bottom
+        anchors.topMargin: -centerRect.height * 0.30
+        anchors.left: centerRect.right
+        anchors.leftMargin: -centerRect.width * 0.20
+        color: alarmColor
+        radius: 30
+        rotation: -50
+        z: 2
+        width: centerRect.width * 0.16
+        height: centerRect.height * 0.25
+    }
+
+    Rectangle {
+        id: leftBottomRect
+        anchors.top: centerRect.bottom
+        anchors.topMargin: -centerRect.height * 0.30
+        anchors.right: centerRect.left
+        anchors.rightMargin: -centerRect.width * 0.20
+        color: alarmColor
+        radius: 30
+        rotation: 50
+        z: 2
+        width: centerRect.width * 0.16
+        height: centerRect.height * 0.25
+    }
+
+    Rectangle {
+        id: centerRect
+        anchors.top: parent.top
+        anchors.topMargin: Kirigami.Units.largeSpacing + Kirigami.Units.smallSpacing
+        anchors.horizontalCenter: parent.horizontalCenter
+        color: "black"
+        radius: width
+        width: Math.min(Math.round(parent.width / 2.1050), 800)
+        height: width
+
+        Rectangle {
+            id: centerRectInner
+            anchors.centerIn: parent
+            color: alarmColor
+            radius: width
+            width: Math.round(parent.width / 1.1180)
+            height: width
+            z: 5
+
+            Item {
+                anchors.fill: parent
+
+                Label {
+                    id: timeI
```

Comment:
Long times like "10:00" don't seem to fit completely inside the clock.

Reply:
should be fixed in recent commit